widgetsnbextension/src/widget_output.js

```diff
@@ -66,7 +66,7 @@ var OutputView = widgets.DOMWidgetView.extend({
     },
 
     render: function(){
-        let renderOutput = (outputArea) => {
+        var renderOutput = function(outputArea) {
             this.output_area = new outputArea.OutputArea({
```

Comment:
Oh, right. Thanks. I tried with the next commit, but I'm already pretty far out of my comfort zone so please tell me if I need to make other changes